EdFi.Buzz.UI.Angular/src/app/Features/UploadSurvey/uploadSurvey.tsx

```diff
@@ -0,0 +1,415 @@
+                        ? 'alert-warning'
+                        : (!fileStatusMessage
+                          || (!fileStatusMessage.error
+                            && fileStatusMessage.serverJobStatus
+                            && fileStatusMessage.serverJobStatus.jobstatuskey === 3))
+                          ? 'alert-success' : ''}>
+                      <h3>File Upload <strong>{fileStatusMessage.status}</strong></h3>
+                      File: {fileStatusMessage.fileName}<br />
+                      {fileStatusMessage.serverJobStatus
+                        ?
+                        <Fragment>
+                          <h3>Job is <strong>{fileStatusMessage.serverJobStatus.jobstatus.description}</strong></h3>
+                          Job Key: {fileStatusMessage.serverJobStatus.jobkey}<br />
+                          {fileStatusMessage.serverJobStatus.resultSummaryObj
+                            ?
+                            <Fragment>
+                              <h3>Job Results</h3>
+                                Survey Key:{fileStatusMessage.serverJobStatus.resultSummaryObj.survey.surveykey}<br />
+                                Survey Title: {fileStatusMessage.serverJobStatus.resultSummaryObj.survey.title}<br />
+                                Questions: {fileStatusMessage.serverJobStatus.resultSummaryObj.survey.questions}<br />
+
+                                Answers Loaded: {fileStatusMessage.serverJobStatus.resultSummaryObj.process.load}<br />
+                                Answers Rejected: {fileStatusMessage.serverJobStatus.resultSummaryObj.process.rejected}<br />
+                                Answers Already Loaded: {fileStatusMessage.serverJobStatus.resultSummaryObj.process.alreadyLoaded}<br />
+                            </Fragment>
+                            : null
+                          }
+                          {(!fileStatusMessage.serverJobStatus.resultSummaryObj && fileStatusMessage.serverJobStatus.resultsummary)
+                            ?
+                            <div className='alert'>
+                              {fileStatusMessage.serverJobStatus.resultsummary}
+                            </div>
+                            : null
+                          }
+                        </Fragment>
+                        : null
+                      }
+                      {fileStatusMessage.error
+                        ?
+                        <p>{fileStatusMessage.error}</p>
+                        : null
+                      }
+                    </div>
+                    : null}
+                </form>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className='row'>
+          <div className='col'>
+            <div className='card-body alert alert-primary text-justify' lang='en'
+              style={{ hyphens: 'auto', marginTop: '-1rem', paddingBottom: '0' }}>
```

Comment:
Consider using a styled-component here to pull our the style. 
Styled components are React component with just the style in CSS. 
So you could create a new style constant inside the `UploadSurvey` object: 

```javascript
const StyledAlertCard = styled.div`
    hyphens: auto;
    margin-top: -1rem; 
    padding-bottom: 0;
`;
```

Then change the div to use the div you've styled as ...
 
```html
            <StyledAlertCard className='card-body alert alert-primary text-justify' lang='en'>
              <p>One column in the survey results must be the unique student identifier and must be called "StudentUniqueId".
        </p>
              <p>Supports Google Forms and Survey Monkey formatted exports. You can also use Qualtrics survey exports by
          removing the first of the two headers.</p>
```

Reply:
Styles will be updated in FIF-242